src/condor_ce_host_network_check

```diff
@@ -47,13 +79,24 @@ class struct_sockaddr_in6(Structure):
 
 
 class union_ifa_ifu(Union):
+    """Anonymous union used for field ifa_ifu in struct ifaddrs from ifaddrs.h
+    See getifaddrs in manpage section 3
+
+    """
     _fields_ = [
         ('ifu_broadaddr', POINTER(struct_sockaddr)),
         ('ifu_dstaddr', POINTER(struct_sockaddr)),]
 
 
 class struct_ifaddrs(Structure):
-    pass
+    """struct ifaddrs from ifaddrs.h
+    See getifaddrs in manpage section 3
+
+    A linked list; each element describes one network interface.  The ifa_next
+    field is a pointer to the next entry in the list.
+
+    """
+# _fields_ set separately because we need union_ifa_ifu to exist first
```

Comment:
OK, then I'm confused. We don't set `_fields_` separately in any of the other structs first; what makes this one different?